npm-shrinkwrap.json

```diff
@@ -8614,4 +8614,4 @@
       "resolved": "https://registry.npmjs.org/yauzl/-/yauzl-2.6.0.tgz"
     }
   }
-}
\ No newline at end of file
+}
```

Comment:
Purely out of curiosity, why is the file-end changing here, when no related change seems to have been made to clean-shrinkwrap.js ?

Reply:
Because I forgot to `+ '\n'` to the JSON output :D